layers/gpu/descriptor_validation/gpuav_descriptor_validation.cpp

```diff
@@ -135,57 +188,69 @@ void UpdateBoundDescriptors(Validator &gpuav, CommandBuffer &cb_state, VkPipelin
     if (!need_descriptor_checks) return true;
 
     for (auto &descriptor_command_binding : cb_state.descriptor_command_bindings) {
-        auto ssbo_block_ptr = (glsl::DescriptorStateSSBO *)descriptor_command_binding.ssbo_block.MapMemory(loc);
+        auto ssbo_block_ptr = (glsl::DescriptorStateSSBO *)descriptor_command_binding.descritpor_state_ssbo_block.MapMemory(loc);
         for (size_t i = 0; i < descriptor_command_binding.bound_descriptor_sets.size(); i++) {
-            DescriptorSet &ds_state = *descriptor_command_binding.bound_descriptor_sets[i].state;
+            DescriptorSet &ds_state = *descriptor_command_binding.bound_descriptor_sets[i];
             ssbo_block_ptr->descriptor_set_types[i] = ds_state.GetTypeAddress(gpuav, loc);
         }
-        descriptor_command_binding.ssbo_block.UnmapMemory();
+        descriptor_command_binding.descritpor_state_ssbo_block.UnmapMemory();
     }
     return true;
 }
 }  // namespace descriptor
 
 // After the GPU executed, we know which descriptor indexes were accessed and can validate with normal Core Validation logic
 [[nodiscard]] bool CommandBuffer::ValidateBindlessDescriptorSets(const Location &loc) {
-    // For each vkCmdBindDescriptorSets()...
-    // Some applications repeatedly call vkCmdBindDescriptorSets() with the same descriptor sets, avoid
-    // checking them multiple times.
-    vvl::unordered_set<VkDescriptorSet> validated_desc_sets;
-    for (auto [command_binding_index, descriptor_command_binding] : vvl::enumerate(descriptor_command_bindings)) {
+    for (uint32_t action_index = 0; action_index < action_command_snapshots.size(); action_index++) {
+        const auto &action_command_snapshot = action_command_snapshots[action_index];
+        const auto &descriptor_command_binding =
+            descriptor_command_bindings[action_command_snapshot.descriptor_command_binding_index];
+
+        // For each vkCmdBindDescriptorSets()...
```

Comment:
```suggestion
        // For each vkCmdBindDescriptorSets()...
```
This comment (this line only) should be titling the enclosing `for` loop